src/scripts/postTransactionBundles.js

```diff
@@ -0,0 +1,81 @@
+const axios = require('axios');
+const path = require('path');
+const fs = require('fs');
+const { createPatientGroupsPerMeasure } = require('../util/groupUtils');
+const mongoUtil = require('../util/mongo');
+
+// for each group, include the patients that are members of the previous group
+// (ex. creates a group of size 10, a group of size 100, ..., a group of size 20,000)
+const GROUP_SIZES = [10, 90, 900, 9000];
+
+/**
+ * Uploads all transaction bundles from the specified directory
+ * into the database. Creates a group resource containing
+ * all the patients (with their updated ids that are created
+ * during the transaction bundle upload process) in the process.
+ * Uploads the created group resource to the database.
+ *
+ * Note: Synthea creates batch bundles for hospital/practitioner info, which
+ * should be POSTed before POSTing the transaction bundles that refer to patients.
+ */
+async function main() {
+  await mongoUtil.client.connect();
+  console.log('Connected successfully to server');
+
+  const bundleDir = path.resolve(process.argv[2]);
+
+  const patientRegEx = new RegExp('Patient/[^/]*');
+  // store uploaded patientIds to be added as members to FHIR Group (across all patients uploaded so far)
+  const allPatientIds = [];
+
+  for (const groupSize of GROUP_SIZES) {
```

Comment:
Will uploads from Synthea always have 10, 90, 900, and 9000 sized groups?

Reply:
Generally they are not restricted to these sizes but for the script itself they will always be this size. I figure this isn't worth making an input argument right now, but we could make it one in the future if needed